bookkeeper-server/src/main/java/org/apache/bookkeeper/replication/Auditor.java

```diff
@@ -1230,7 +1266,23 @@ void checkAllLedgers() throws BKException, IOException, InterruptedException, Ke
                     return;
                 }
 
+                try {
+                    if (!openLedgerNoRecoverySemaphore.tryAcquire(openLedgerNoRecoverySemaphoreWaitTimeoutMSec,
```

Comment:
I think openLedgerNoRecoverySemaphoreWaitTimeoutMSec can be safely set to zk client timeout, or 2x of that timeout

Reply:
The default zk client timeout is 10s, if set to 2 * zkTimeout, it will be 20s. 
However, in the replicas check process, it use `REPLICAS_CHECK_TIMEOUT_IN_SECS=120` as maxConcurrentSemaphore tryAcquire timeout.
So i doubt whether put `openLedgerNoRecoverySemaphoreWaitTimeoutMSec` in to `bk_server.conf` or just hard code to `2 * zkTimeout` and the default value set to 120s or not. 
@dlg99  Would you please give me some ideas?

Reply:
@hangc0276 ouch. let's just leave it as it is.